scripts/base/build_python.sh

```diff
@@ -14,42 +23,43 @@ export LD_LIBRARY_PATH=/opt/rh/devtoolset-6/root/usr/lib64:/opt/rh/devtoolset-6/
 export PATH=/opt/rh/devtoolset-6/root/usr/bin:/opt/app-root/src/bin:/opt/rh/devtoolset-6/root/usr/bin/:/usr/sbin:/usr/bin:/sbin:/bin
 exec "\$@"
 EOF
-chmod a+x /usr/local/bin/run-with-system-python
+chmod a+x ${ASWF_INSTALL_PREFIX}/bin/run-with-system-python
 
 # Create a yum wrapper that uses the system python
-cat <<EOF > /usr/local/bin/yum
+cat <<EOF > ${ASWF_INSTALL_PREFIX}/bin/yum
 #!/bin/sh
 # This runs yum with system python
-exec /usr/local/bin/run-with-system-python /usr/bin/yum "\$@"
+exec ${ASWF_INSTALL_PREFIX}/bin/run-with-system-python /usr/bin/yum "\$@"
 EOF
-chmod a+x /usr/local/bin/yum
+chmod a+x ${ASWF_INSTALL_PREFIX}/bin/yum
 
-curl --location https://www.python.org/ftp/python/${PYTHON_VERSION_FULL}/Python-${PYTHON_VERSION_FULL}.tgz -o Python.tgz
-tar xf Python.tgz && rm Python.tgz
+tar xf $DOWNLOADS_DIR/Python-${PYTHON_VERSION_FULL}.tgz
 cd Python-${PYTHON_VERSION_FULL}
 
-# Ensure configure, build and install is done with no reference to /usr/local as this somehow messes up the system install
+# Ensure configure, build and install is done with no reference to ${ASWF_INSTALL_PREFIX} as this somehow messes up the system install
 run-with-system-python ./configure \
-    --prefix=/usr/local \
+    --prefix=${ASWF_INSTALL_PREFIX} \
     --enable-unicode=ucs4 \
     --enable-shared
 run-with-system-python make -j4
 
-run-with-system-python make altinstall
+run-with-system-python make install
 
 if [[ $PYTHON_VERSION == 3* ]]; then
     # Create symlink from python3 to python
-    sudo ln -s /usr/local/bin/python3 /usr/local/bin/python
+    ln -s python3 ${ASWF_INSTALL_PREFIX}/bin/python
 fi
 
-cd ..
-rm -rf Python-${PYTHON_VERSION_FULL}
+cd ../..
+rm -rf python
 
-curl "https://bootstrap.pypa.io/get-pip.py" -o "get-pip.py" &&\
-    python get-pip.py
+curl "https://bootstrap.pypa.io/get-pip.py" -o "get-pip.py"
+python get-pip.py
 rm get-pip.py
 
 pip install \
```

Comment:
In the previous 2020 image, although only Python3 was installed so in theory pip should have been a softlink to pip3, the OpenEXR builds didn't find numpy. We can leave this as is and try the builds first to see if it works. Or could change to call pip3 directly:

if [[ $PYTHON_VERSION == 3* ]]; then
    export PIP=pip3
else
   export PIP=pip
fi

exec $PIP install \